recipes/r-rnpn/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set version = '1.2.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rnpn
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rnpn_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rnpn/rnpn_{{ version }}.tar.gz
+  sha256: 0900bff533e5ee281b309a91c307c6b19482f2a60136e7d8e84ba7da56346a0a
+
+build:
+  noarch: generic
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-curl
+    - r-data.table >=1.9.6
+    - r-httr >=1.1.0
+    - r-jsonlite >=0.9.19
+    - r-magrittr
+    - r-plyr
+    - r-raster
+    - r-rgdal
+    - r-sp >=1.1-0
+    - r-tibble
+    - r-xml
+
+  run:
+    - r-base
+    - r-curl
+    - r-data.table >=1.9.6
+    - r-httr >=1.1.0
+    - r-jsonlite >=0.9.19
+    - r-magrittr
+    - r-plyr
+    - r-raster
+    - r-rgdal
+    - r-sp >=1.1-0
+    - r-tibble
+    - r-xml
+
+test:
+  commands:
+    - $R -e "library('rnpn')"           # [not win]
+    - "\"%R%\" -e \"library('rnpn')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/package=rnpn
+  license: MIT
+  summary: Programmatic interface to the Web Service methods provided by the National 'Phenology'
+    Network (<https://usanpn.org/>), which includes data on various life history events
+    that occur at specific times.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - thewchan
+
+# The original CRAN metadata for this package was:
+
+# Package: rnpn
+# Title: Interface to the National 'Phenology' Network 'API'
+# Description: Programmatic interface to the Web Service methods provided by the National 'Phenology' Network (<https://usanpn.org/>), which includes data on various life history events that occur at specific times.
+# Version: 0.1.0
```

Comment:
Why is the attached metadata so ancient? How was the recipe generated? (we may want to document this as a bug)

Reply:
hmm it's a great question, let me try again

Reply:
I might have pulled an older version by mistake; I did noticed some discrepancies so I updated manually, let me regenerating again